include/cpuinfo_ppc.h

```diff
@@ -0,0 +1,142 @@
+// Copyright 2018 IBM
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef CPU_FEATURES_INCLUDE_CPUINFO_PPC_H_
+#define CPU_FEATURES_INCLUDE_CPUINFO_PPC_H_
+
+#include "cpu_features_macros.h"
+#include "internal/hwcaps.h"
+
+#define MAX_LINE_SIZE 100
```

Comment:
This will leak out of the header and possibly conflict (or replace) other definitions.
Just set the number in the struct directly.

Also 100 is probably too much. What would be a good size here? 32? 64?

Reply:
Good point, I'll do that and use 64.